tables/endpointsecurity/src/processeventstableplugin.cpp

```diff
@@ -136,6 +136,21 @@ Status ProcessEventsTablePlugin::generateRow(
 
   row = {};
 
+  std::string action;
+
+  switch (event.type) {
+  case IEndpointSecurityConsumer::Event::Type::Exec:
+    action = "exec";
+    break;
+
+  case IEndpointSecurityConsumer::Event::Type::Fork:
+    action = "fork";
+    break;
+
+  default:
```

Comment:
Let's enumerate the cases we ignore explicitly so we get notified here should new enum values be added in the future.